src/blog/2024/05/mapping-location-on-dashboard-2.md

```diff
@@ -0,0 +1,116 @@
+
+![Screenshot of the Change node setting the icon and icon color for bus](./images/mapping-location-on-dashboard-2-change-node-setting-icon-for-bus.png "Screenshot of the Change node setting the icon and icon color for bus"){data-zoomable}
+
+8. Connect the **HTTP request** node's output to the input of the **Change** node named **Set payload**, and connect the output of that **Change** node to the input of the **Split** node.
+9. Then, connect the output of the **Split** node to the input of the **Change** node named **Change and delete properties**, and connect the output of the "Change and delete properties" node to the input of the **Switch** node. Then **Switch** node's first output to the input of the **Change** node named **set icon and icon color for tram**, and its second output to the input of the **Change** node named **set icon and icon color for bus**.
+ 
+## Plotting location data on Worldmap
+
+1. Drag a **Worldmap** node onto the canvas. Set the path to **/worldmap** and keep the rest of the settings unchanged, although you can modify other properties according to your preferences.
+
+!["Screenshot displaying the configuration of the Worldmap custom node"](./images/mapping-location-on-dashboard-2-worldmap-node.png "Screenshot displaying the configuration of the Worldmap custom node"){data-zoomable}
+
+2. With the **worldmap** node configured, it will generate a world map with plotted data accessible at the specified path.
+3. Connect the **Function** node's output to **Worldmap** node's input.
+
+Now we have successfully created a page with a world map displaying plotted vehicle location data. To confirm, you can visit `https://<your-instance-name>.flowfuse.cloud/worldmap`.
+
+## Rendering map on Dashboard 2.0
+
+To render worlmap webpage on dashboard 2.0 we will use **iframe** custom widget which will allow us to embed an external webpage into Dashboard 2.0 applications using an iframe.
+
+### Installing iframe custom widget
+
+1. Click the Node-RED Settings (top-right)
+2. Click **Manage Palette**
+3. Switch to the **Install** tab
+4. Search for `@flowfuse/node-red-dashboard-2-ui-iframe`
+5. Click **Install**
+
+### Rendering worlmap on Dashboard 2.0
+
+1. Drag an **iframe** widget onto the canvas.
+2. Select **ui-group** and **ui-base** for it, where you want to render the webpage.
+3. Set height and width for it according to your preference and set URL to **/worlmap**.
+
+!["Screenshot showing configurations of iframe widget for rendering worlmap page on dashboard"](./images/mapping-location-on-dashboard-2-iframe-widget.png "Screenshot showing configurations of iframe widget for rendering worlmap page on dashboard"){data-zoomable}
+
+## Deploying the flow
+
+!["Image displaying live locations of UK public transport on the dashboard"](./images/mapping-location-on-dashboard-2-uk-live-transport.gif "Image displaying live locations of UK public transport on the dashboard"){data-zoomable}
```

Comment:
Follow up to include meta data? E.g. can you click on a bus to show some additional data?